test/quantization/fx/test_equalize_fx.py

```diff
@@ -110,3 +124,48 @@ def test_input_weight_observer(self, input_qdtype, input_qscheme, weight_qdtype,
             ref_scales, dtype=weight_qparams[0].dtype), atol=0.0001))
         self.assertTrue(torch.allclose(weight_qparams[1], torch.tensor(
             ref_zero_points, dtype=weight_qparams[1].dtype), atol=1))
+
+    def test_input_weight_equalization_prepare(self):
```

Comment:
Can we also add a couple different models to this test
1. test nn.functional.linear
2. Linear followed by fp32 op
2. Linear followed by another quantized op (non-linear)